app/models/admin/scholarship_group.rb

```diff
@@ -1,4 +1,5 @@
 class Admin::ScholarshipGroup < ApplicationRecord
   validates_presence_of :name
   validates_uniqueness_of :name
+  has_many :admin_scholarships, :class_name => 'Admin::Scholarship',inverse_of: :admin_scholarship_group
```

Comment:
Why does require to add this `has_many :admin_scholarships, :class_name => 'Admin::Scholarship',inverse_of: :admin_scholarship_group` ?